src/confcom/azext_confcom/README.md

```diff
@@ -766,6 +824,29 @@ az confcom acifragmentgen --chain ./samples/certs/intermediateCA/certs/www.conto
 
 This could be useful in scenarios where an image-attached fragment is required but the fragment's feed is different from the image's location.
 
+Example 5: This format can also be used to generate fragments used for VN2. Adding the `scenario` key with the value `vn2` tells confcom which default values need to be added. Save this file as `fragment_config.json`:
```

Comment:
This README has gotten really long. We should consider splitting it up into separate readmes per example/scenario/use